contracts/core/fee-calculators/PerformanceFeeCalculator.sol

```diff
@@ -0,0 +1,377 @@
+/*
+    Copyright 2020 Set Labs Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+pragma solidity 0.5.7;
+
+import { SafeMath } from "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+import { CommonMath } from "../../lib/CommonMath.sol";
+import { ICore } from "../interfaces/ICore.sol";
+import { IFeeCalculator } from "../interfaces/IFeeCalculator.sol";
+import { IOracleWhiteList } from "../interfaces/IOracleWhiteList.sol";
+import { IRebalancingSetTokenV2 } from "../interfaces/IRebalancingSetTokenV2.sol";
+import { ISetToken } from "../interfaces/ISetToken.sol";
+import { ScaleValidations } from "../../lib/ScaleValidations.sol";
+import { SetUSDValuation } from "../liquidators/impl/SetUSDValuation.sol";
+
+
+/**
+ * @title PerformanceFeeCalculator
+ * @author Set Protocol
+ *
+ * Smart contract that stores and returns fees (represented as scaled decimal values). Fees are
+ * determined based on performance of the Set and a streaming fee. Set values can be denominated
+ * in any any asset based on oracle white list used in deploy.
+ */
+contract PerformanceFeeCalculator is IFeeCalculator {
+
+    using SafeMath for uint256;
+    using CommonMath for uint256;
+
+    /* ============ Events ============ */
+
+    event FeeActualized(
+        address indexed rebalancingSetToken,
+        uint256 newHighWatermark,
+        uint256 inflationFee
```

Comment:
In addition to inflationFee, I think it would be useful to print out what the `streamingFee` and `profitFee`. How else would the backend know what to use? Or otherwise, it would need to be implied.